src/codegeneration/CommonJsModuleTransformer.js

```diff
@@ -43,7 +43,7 @@ import {prependStatements} from './PrependStatements.js';
 
 export class CommonJsModuleTransformer extends ModuleTransformer {
 
-  constructor(identifierGenerator, reporter, options) {
+  constructor(identifierGenerator, reporter, options = undefined) {
```

Comment:
I asked for this. I would have preferred that we tracked down the invalid caller but I did not want to put that burden on @guybedford